src/build/docker/debian/Dockerfile

```diff
@@ -10,6 +10,7 @@ RUN DEBIAN_FRONTEND="noninteractive" apt-get -qq update && \
     libonig-dev \
     lintian \
     lsb-release \
+    libxml2-utils \
```

Comment:
Why is this here?

Reply:
`xmllint` is needed when compiling builder ui files into resources (for the `xml-stripblanks` operation).